### ISSUE TITLE
Dear PAX... Re: accessibility

### ISSUE THREAD

**Issue by ryanve** [ryanve/a11y#5]
> I was browsing [paxvapor.com](https://paxvapor.com) and realized a major accessibility flaw that your web team can easily fix. I'm considering buying a PAX Era but I feel conflicted about doing so until this flaw is corrected. The flaw is that if you tab thru the website with your keyboard there's no visual distinction of which element is focused. For website users unable to use a mouse this makes the site nearly impossible to navigate. Browsers provide an outline for this by default but the CSS on [paxvapor.com](https://paxvapor.com) is removing that outline without providing any alternative distinction. Resources about this can be found on http://www.outlinenone.com
> 
> I would encourage PAX to make its website and products accessible and inclusive. PAX seems to care about design and being accessible only increases your potential user base. Other innovative companies such as Apple and Google have prioritized accessibility to obvious success. Web accessibility standards are detailed on https://www.w3.org/TR/WCAG/
